Automattic/wp-calypso
# Reader: Use intro banner with white background as default

After running the intro AB test in: https://github.com/Automattic/wp-calypso/pull/16330, results show that there is a lack of statistical significance. So this PR:

1. Removes code and files associated with the AB test.
2. Uses option (C) https://github.com/Automattic/wp-calypso/pull/16330 as the default intro banner for everyone:

<img width="786" alt="screenshot 2017-09-25 19 01 19" src="https://user-images.githubusercontent.com/4924246/30839019-f4cf4920-a223-11e7-9a16-732035c769e3.png">

## To test

Sign up for a brand new WordPress.com account, navigate to http://calypso.localhost:3000 and verify that you see the white intro banner above.

/cc @bluefuton @aaronyan
